spring-integration-file/src/main/java/org/springframework/integration/file/config/AbstractRemoteFileInboundChannelAdapterParser.java

```diff
@@ -49,7 +51,8 @@ protected final BeanMetadataElement parseSource(Element element, ParserContext p
 
 		// configure the InboundFileSynchronizer properties
 		BeanDefinition expressionDef = IntegrationNamespaceUtils.createExpressionDefinitionFromValueOrExpression(
-				"remote-directory", "remote-directory-expression", parserContext, element, false);
+				"remote-directory", "remote-directory-expression", parserContext,
+				element, false);
```

Comment:
Doh...
In new IDEA it looks like:
![idea_120](https://cloud.githubusercontent.com/assets/1071590/23620542/4c78555e-0265-11e7-9ac9-850b9d495d54.png)

The visual perception deceived me... 😢 

My bad, sorry.
Will be much careful in the future.
Reverting...